operator/k8s_identity.go

```diff
@@ -64,12 +65,21 @@ func deleteIdentity(ctx context.Context, identity *v2.CiliumIdentity) error {
 
 var identityHeartbeat *identity.IdentityHeartbeatStore
 
+// counters for GC failed/successful runs
+var (
+	failedRuns     = 0
+	successfulRuns = 0
+)
+
 // identityGCIteration is a single iteration of a garbage collection. It will
 // delete identities that have not had its heartbeat lifesign updated since
 // option.Config.IdentityHeartbeatTimeout
 func identityGCIteration(ctx context.Context) {
 	log.Debug("Running CRD identity garbage collector")
 
+	totalEntries := len(identityStore.List())
```

Comment:
`identityStore.List()` is also retrieved on line 94. To avoid executing this function 2 times, because it's costly, consider setting a variable with the output of `List()` and use the variable to calculate the number of elements in the list as well as to use it in the for loop, istead of using identityStore.List()`, on line 94.